### ISSUE TITLE
Tutorial Genetic algorithm fails

### ISSUE THREAD

**Issue by teobros** [cossan-working-group/OpenCossan#8]
The tutorial of genetic algorithm fails the error "fail to evaluate fitness function". Discovered by EP, needs to be addressed by someone...

**Comment by JCSadeghi** [comment]
I see the following error
```
Error using GeneticAlgorithms (line 188)
PropertyName toleranceDesignVariables not valid

Error in TutorialGeneticAlgorithms (line 183)
Xga     = GeneticAlgorithms('NPopulationSize',10,...
 ```

Running in Matlab R2017a.

Can you confirm it is the same bug? Happy to have a look at this if nobody else is already working on it. 

(If someone else is already working on this please assign yourself to the issue)

**Comment by etheban** [comment]
I have already fixed the toleranceDesignVariables. I will now commit

**Comment by JCSadeghi** [comment]
The error to fix in this issue is as follows:
```
In an assignment  A(:) = B, the number of elements in A and B must be the same.

Error in augLagFun (line 132)
fval(to_evaluate) = feval(objfun,points,objFcnArg{:});

Error in createAnonymousFcn>@(x)fcn(x,FcnArgs{:}) (line 11)
fcn_handle = @(x) fcn(x,FcnArgs{:});

Error in makeState (line 64)
            Score = FitnessFcn(state.Population(initScoreProvided+1:end,:));

Error in gacon (line 39)
state = makeState(GenomeLength,SubFitness,Iterate,subtype,options);

Error in ga (line 378)
                [x,fval,exitFlag,output,population,scores] = gacon(FitnessFcn,nvars, ...

Error in GeneticAlgorithms/apply (line 197)
        [Vsol,Vfval,Nexitflag, ~] =ga(hobjfun,...

Error in TutorialGeneticAlgorithms (line 189)
Xoptimum  = Xga.apply('XoptimizationProblem',Xop,'LplotEvolution',false);

Caused by:
    Failure in initial user-supplied fitness function evaluation. GA cannot continue.
```

**Comment by JCSadeghi** [comment]
I think the problem is caused by attempting to define two objective functions and pass them in a cell array:
```
%% Define the fittness (i.e. Objective Function)
Xobj=ObjectiveFunction('Sdescription','simple fitness', ...
    'Lfunction',false,'Liomatrix',true,'Liostructure',false,...
    'Cinputnames',{'X1','X2'},... % Define the inputs
    'Sscript','Moutput = 100*(Minput(:,1).^2 - Minput(:,2)).^2 + (1 - Minput(:,1)).^2;',... % the real function
    'Coutputnames',{'fobj'}); % Define the outputs

Xobj2=ObjectiveFunction('Sdescription','simple fitness', ...
    'Lfunction',false,'Liomatrix',true,'Liostructure',false,...
    'Cinputnames',{'X1','X2'},... % Define the inputs
    'Sscript','Moutput = 100*(Minput(:,1).^2 - Minput(:,2)).^2 + (1 - Minput(:,1)).^2;',... % the real function
    'Coutputnames',{'fobj2'}); % Define the outputs
```

**Comment by JCSadeghi** [comment]
Fixed in #10 - the tutorial now runs correctly.
It might also be worth adding some validation to the genetic algorithm to prevent OptimizationProblems with multiple objective functions being evaluated. I'll open a separate issue for that.

**Comment by JCSadeghi** [comment]
Closing as issue is fixed, but we should resolve #11 to prevent this reoccuring
